Source/MillicastPublisher/Private/WebRTC/Texture2DFrameBuffer.h

```diff
@@ -5,22 +5,7 @@
 #include "WebRTCInc.h"
 #include "RHI.h"
 #include "RHIGPUReadback.h"
-
```

Comment:
We can remove this extern and just include normally due to the change to build.cs